frontend/packages/kubevirt-plugin/src/selectors/immutable/template/combined.ts

```diff
@@ -97,3 +98,8 @@ export const iGetRelevantTemplates = (
 export const iGetRelevantTemplate = (
   ...args: Parameters<typeof iGetRelevantTemplates>
 ): ITemplate => iGetRelevantTemplates(...args).first();
+
+export const iGetCommonTemplateCloudInit = (tmp: ITemplate) =>
```

Comment:
please make an immutable version of `selectVM`. Specifying an index is unreliable.

Reply:
created